aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -1703,15 +1714,16 @@ static Tensor& _linalg_norm_vector_out(Tensor& result, const Tensor& self, optio
       // size is degenerate (like size(0), size(0, N), etc)
       case_was_overridden = true;
       self_ = self_.abs();
-      result_ = _norm_min_max(self_, ord, dim[0], keepdim);
+      result_ = _norm_min_max(self_, ord, dim[0], keepdim, flatten);
     } else if ((self_.numel() == 0) && (ord < 0)) {
       // For negative orders with degenerate input sizes, at::norm's result does not
       // match numpy. It should always be infinity.
-      auto mask = make_dim_mask(dim[0], self_.dim());
+      // auto mask = make_dim_mask(flatten ? dim : self.dims(), self_.dim());
```

Comment:
This comment is looks like some left over code?

A comment explaining how flatten affects the mask here would be helpful.